### ISSUE TITLE
@telus/create-library package published on npmjs is broken due to missing dependency

### ISSUE THREAD

**Issue by pdufault** [telus/create-library#68]
I'm not able to leverage this initializer as the docs describe:
```
$ npm init @telus/library my-new-project
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@telus%2fcore - Not found
npm ERR! 404
npm ERR! 404  '@telus/core@^0.0.3' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of '@telus/create-library'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/phildufault/.npm/_logs/2019-12-30T18_59_33_802Z-debug.log
Install for @telus/create-library@latest failed with code 1
```

I'm following https://github.com/telus/create-library#-usage

The npm package published seems stale, as in the Git repo @telus/core is not referenced.

This might be a quick win, to publish a new release of this package to npmjs.org

I was able to clone this package's repo and use it locally to initialize a new npm library repo, so I believe just a new release to npmjs.org is needed.

Cheers! ❤️

**Comment by pdufault** [comment]
🎉 It is working now, thanks @maumercado 

```$ npm init @telus/library my-new-project
npx: installed 84 in 4.526s
? is this an open source project? (y/N) y```

**Comment by pdufault** [comment]
(fixed in #65)
